tests/test-helper.js

```diff
@@ -2,6 +2,29 @@ import Application from '../app';
 import config from '../config/environment';
 import { setApplication } from '@ember/test-helpers';
 import { start } from 'ember-qunit';
+import QUnit from 'qunit';
+import { __counts } from 'ember-on-modifier/modifiers/on';
+
+let startingCounts;
+QUnit.testStart(() => {
+  startingCounts = __counts();
```

Comment:
`testStart` has no `this` context, sadly. The alternative would've been a `setupAssertCounts(hooks)` test helper that adds an `assertCount` util to `this`, but I think this is still a bit cleaners.

What do you think?

Reply:
```suggestion
  QUnit.config.current.testEnvironment._startingCounts = __counts();
```

`QUnit.config.current.testEnvironment ` represents the `this` context of the test